src/averageAges.js

```diff
@@ -20,6 +20,14 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let mens = people.filter(person => person.sex === 'm');
+
+  /* eslint-disable max-len */
+  mens = century ? mens.filter(men => Math.ceil(men.died / 100) === century) : mens;
+
+  const yearsAlive = mens.map(men => men.died - men.born);
+
+  return yearsAlive.reduce((sum, year) => sum + year, 0) / yearsAlive.length;
```

Comment:
Could you combine `map` and `reduce` ?

You could calculate the age difference inside reduce (`(sum, person) => sum + person.died - person.born `)